### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

**Comment by loganfsmyth** [comment]
Looks awesome.

**Comment by hzoo** [comment]
Happy to hear - I guess I did learn some things this year 😄 , should of started this earlier

**Comment by thejameskyle** [comment]
I forgot to submit my review earlier

File: translations/en/plugin-handbook.md [hunk 1]
```diff
@@ -390,6 +396,11 @@ const MyVisitor = {
     console.log("Called!");
   }
 };
+
+// You can also create a visitor and add methods on it later
+let visitor = {};
+visitor.MemberExpression = function() {};
+visitor.FunctionDeclaration = function() {}
```
Comment (hzoo) [inline translations/en/plugin-handbook.md]: Not necessary just writing how it's done (and yeah I guess it's obvious since it's an object)

**Pull request merged**